src/main/java/worldofzuul/presentation/WindturbineShopController.java

```diff
@@ -0,0 +1,76 @@
+package worldofzuul.presentation;
+
+import javafx.event.ActionEvent;
+import javafx.fxml.FXML;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.Parent;
+import javafx.scene.Scene;
+import javafx.scene.control.Button;
+import javafx.scene.control.Label;
+import javafx.scene.image.ImageView;
+import javafx.scene.input.MouseEvent;
+import javafx.stage.Stage;
+import worldofzuul.Game;
+import worldofzuul.Input.Command;
+import worldofzuul.Input.CommandWord;
+import worldofzuul.Rooms.Shops.Shop;
+
+import java.io.IOException;
+import java.net.URL;
+import java.net.URLDecoder;
+import java.nio.charset.StandardCharsets;
+
+public class WindturbineShopController {
+    @FXML
+    Label windPrice1, windPrice2;
+    @FXML
+    Button btnHouse;
+    @FXML
+    private URL location;
+
+    @FXML
+    public void initialize() {
+        Shop foundShop = null;
+        if (windPrice1 != null) {
```

Comment:
Is this check necessary?

Reply:
**THIS JUST IN... (BREAKING NEWS)**
It wasn't necessary, just checked :)